metaswitch/common/utils.py

```diff
@@ -401,6 +401,29 @@ def sigusr1_handler(sig, stack):
         write_core_file(process_name, stack_dump)
     signal.signal(signal.SIGUSR1, sigusr1_handler)
 
+def map_clearwater_log_level(level):
+    """
+    Map from Clearwater log levels to Python log levels.
+
+    Python doesn't have status or verbose levels, so these are mapped to
+    info and debug respectively.
+    """
+    LOG_LEVELS = {0: logging.ERROR,
+                  1: logging.WARNING,
+                  2: logging.INFO,
```

Comment:
I'll add a flag - info_is_normal to control whether this is WARNING or INFO - as we differ in opinion between cedar-ftpd and crest/etcd.